bot/cogs/cyber.py

```diff
@@ -369,6 +374,20 @@ def suffix_from_number(num):
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
 
+        # Level Function Regex.
+        match = self.game_level_regex.match(message.content)
+        if match and message.channel.id in [HQ_ID, MOON_ID, FORENSICS_ID]:
+            level_num, challenge_num = int(match.group(2)), int(match.group(3))
+            if message.channel.id == HQ_ID:
+                base = 'hq'
+            elif message.channel.id == MOON_ID:
+                base = 'm'
+            elif message.channel.id == FORENSICS_ID:
+                base = 'f'
+            message.content = f"{self.bot.user.mention} level {base} {level_num} {challenge_num}"
+            new_ctx = await self.bot.get_context(message)
```

Comment:
```suggestion
            ctx.invoke(self.bot.get_command('level'), base, level_num, challenge_num)
```
slightly less hacky way of doing this.